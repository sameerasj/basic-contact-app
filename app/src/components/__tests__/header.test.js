import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Header from "../Header";

it("should render header", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.find("h2").text()).toEqual("Contact Manager");
});

it("should center header title", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.find({ "data-testid": "menu" }).props().className).toContain(
    " center"
  );
});

// Snapshot test
it("should match header snapshot", () => {
  const snap = renderer.create(<Header />).toJSON();
  expect(snap).toMatchSnapshot();
})
