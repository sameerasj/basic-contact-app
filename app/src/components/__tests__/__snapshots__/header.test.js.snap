// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match header snapshot 1`] = `
<div
  className="ui fixed menu"
>
  <div
    className="ui container center"
    data-testid="menu"
  >
    <h2>
      Contact Manager
    </h2>
  </div>
</div>
`;
